itest/rust/src/benchmarks/mod.rs

```diff
@@ -93,6 +93,24 @@ fn utilities_ffi_call() -> f64 {
     godot::global::pow(base, exponent)
 }
 
+#[bench(repeat = 25)]
+fn packed_array_from_iter_known_size() -> PackedInt32Array {
+    PackedInt32Array::from_iter(0..100)
+}
+
+#[bench(repeat = 25)]
+fn packed_array_from_iter_unknown_size() -> PackedInt32Array {
+    let mut item = 0;
+    PackedInt32Array::from_iter(std::iter::from_fn(|| {
+        item += 1;
+        if item <= 100 {
+            Some(item)
+        } else {
+            None
+        }
+    }))
```

Comment:
Could we not use a range here, e.g. `(1..=100)`?

It's not clear to me which benefit `from_fn()` adds here.

Reply:
It's commented in the test, but not in the benchmark. Copied the comment here.